.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
 install:
 - npm install
 - pip install --user -r requirements.txt
-- if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="gcc-4.9"; fi
+- if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="g++-4.9 -std=c++11"; fi
```

Comment:
If I'm not mistaken Craig said we need C++11 in order to support asin, acos, and atan on complex numbers, are there other alternatives?